### PR TITLE
Add showcase author metadata

### PR DESCRIPTION
**Pull request by clementbiron**
Add author metadata to showcase and display author name when it is filled.

image

@MattiSG  suggests that we change the icon according to the type of author, I find the idea attractive but it's not very easy to find according icons for `collective|ngo|research_center|business|local_government|national_government` in the [lib used](https://lucide.dev/). If you find icons in this library that you think work, let me know.

### CODE REVIEW

**Comment by MattiSG** [comment]
Not perfect, but I believe this could do:

- `collective` → `users`
- `ngo` → `globe`
- `research_center` → `graduation-cap`
- `business` → `factory`
- `local_government` → `building`
- `national_government` → `landmark`

**Review by MattiSG — APPROVED** [review]
Looks great!

**Comment by clementbiron** [comment]
> Not perfect, but I believe this could do:
> 
>     * `collective` → `users`
> 
>     * `ngo` → `globe`
> 
>     * `research_center` → `graduation-cap`
> 
>     * `business` → `factory`
> 
>     * `local_government` → `building`
> 
>     * `national_government` → `landmark`

Thank you for this suggestion, I have implemented it.

**Comment by MattiSG** [comment]
Are we waiting for anything to merge this? :slight$

**Comment by clementbiron** [comment]
Following our discussion earlier, I am working on it ;)

**Comment by clementbiron** [comment]
It was much more complicated than I thought but I simplified it well, see https://github.com/openfisca/openfisca.org/pull/118/commits/aadaa8baa0bdefc44db5270bcbc4697e53922f5: it allowed me to better understand how context is propagated in partials 🧠

@MattiSG I'm sorry I've had a complicated day and I think it's necessary that you review the PR again, especially with the addition of the authors for each reuse, see https://github.com/openfisca/openfisca.org/pull/118/commits/091d4aa968a47e20775144a5e9a8dc064918c414

File: data/showcase/leximpact.yml
Comment (MattiSG) [inline data/showcase/leximpact.yml:9]:
```suggestion
  name: Assemblée nationale
```

File: data/showcase/mes_aides.yml
Comment (MattiSG) [inline data/showcase/mes_aides.yml:9]:
```suggestion
  name: beta.gouv.fr
```

File: data/showcase/nsw_legislation_explorer.yml
Comment (MattiSG) [inline data/showcase/nsw_legislation_explorer.yml:9]:
```suggestion
  name: New South Wales Government
```

File: data/showcase/rapu_ture.yml
Comment (MattiSG) [inline data/showcase/rapu_ture.yml:9]:
```suggestion
  name: New Zealand Government Innovation Service Lab
```

File: data/showcase/revenu_de_base.yml
Comment (MattiSG) [inline data/showcase/revenu_de_base.yml:9]: Are we limited in the number of characters for the layout? If so, please make sure this is reflected in #121  🙂 
```suggestion
  name: Mouvement Français pour un Revenu de Base
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
I don't understand why Netlify fails when the build passes in CircleCi 🤔

**Comment by MattiSG** [comment]
I'll handle renames 🙂

**Comment by MattiSG** [comment]
I let two suggestions open as I'm not sure if it is a problem for you that it renders in two lines. For myself, I'm okay with it, but I'll let you decide 🙂 
image

**Review by MattiSG — APPROVED** [review]
Nice!! 🚀

File: data/showcase/revenu_de_base.yml
Comment (clementbiron) [reply data/showcase/revenu_de_base.yml]: I have updated the authors' names without character restrictions.

File: data/showcase/rapu_ture.yml
Comment (clementbiron) [reply data/showcase/rapu_ture.yml]: 👍

**Pull request merged**
